e2e/cypress/support/commands.js

```diff
@@ -0,0 +1,178 @@
+});
+
+// Cypress.Commands.add('delete_dataset', (datasetName) => {
+//   /**
+//    * Method to purge a dataset from the current state
+//    * :PARAM datasetName String: Name of the dataset to purge from the current state
+//    * :RETURN null:
+//    */
+//   cy.request({
+//       url: '/api/action/dataset_purge',
+//       method: 'POST',
+//       body: {
+//           id: datasetName,
+//       },
+//   });
+// });
+
+Cypress.Commands.add(
+  'create_harvest_source',
+  (dataSourceUrl, harvestTitle, harvestDesc, harvestType, harvestPrivate, invalidTest) => {
+      /**
+       * Method to create a new CKAN harvest source via the CKAN harvest form
+       * :PARAM dataSourceUrl String: URL to source the data that will be harvested
+       * :PARAM harvestTitle String: Title of the organization's harvest
+       * :PARAM harvestDesc String: Description of the harvest being created
+       * :PARAM harvestType String: Harvest source type. Ex: waf, datajson
+       * :RETURN null:
+       */
+      if (!invalidTest) {
+          cy.get('#field-url').type(dataSourceUrl);
+      }
+      cy.get('#field-title').type(harvestTitle);
+      cy.get('#field-name').then(($field_name) => {
+          if ($field_name.is(':visible')) {
+              $field_name.type(harvestTitle);
+          }
+      });
+
+      cy.get('#field-notes').type(harvestDesc);
+      cy.get('[type="radio"]').check(harvestType);
+
+      // Validate private_datasets defaults to Private
+      cy.get('#field-private_datasets').find(':selected').contains('Private');
+
+      cy.get('#field-private_datasets').select(harvestPrivate);
+
+      cy.get('input[name=save]').click();
+  }
+);
+
+Cypress.Commands.add('delete_harvest_source', (harvestName) => {
+  cy.visit('/harvest/admin/' + harvestName);
+  cy.wait(3000);
+  cy.contains('Clear').click({ force: true });
+
+  // Confirm harvest clear
+  cy.wait(1000);
+  cy.contains(/^Confirm$/).click();
+
+  cy.wait(3000);
+  cy.visit('/harvest/delete/' + harvestName + '?clear=True');
+});
+
+Cypress.Commands.add('start_harvest_job', (harvestName) => {
+  cy.visit('/harvest/' + harvestName);
+  cy.hide_debug_toolbar();
+
+  cy.contains('Admin').click();
+  // Wait for all pages to load, avoid bug
+  // https://github.com/ckan/ckanext-harvest/issues/440
+  cy.wait(3000);
+  cy.get('.btn-group>.btn:first-child:not(:last-child):not(.dropdown-toggle)').click({ force: true });
+  // Confirm harvest start
+  cy.wait(1000);
+  cy.contains(/^Confirm$/).click();
+
+  // Confirm stop button exists, harvest is started/queued
+  cy.wait(500);
+  cy.contains('Stop');
+});
+
+// Cypress.Commands.add('create_dataset', (ckan_dataset) => {
+//   var options = {
+//       method: 'POST',
+//       url: '/api/3/action/package_create',
+//       headers: {
+//           'cache-control': 'no-cache',
+//           'content-type': 'application/json',
+//       },
+//       body: JSON.stringify(ckan_dataset),
+//   };
+
+//   return cy.request(options);
+// });
```

Comment:
Since we're no longer using harvest sources to test, should we remove all this code?

Reply:
yes, just removed those unused files.